compiler/circle-tensordump/CMakeLists.txt

```diff
@@ -2,7 +2,12 @@ if(NOT TARGET mio_circle)
   return()
 endif(NOT TARGET mio_circle)
 
-find_package(HDF5 COMPONENTS CXX QUIET)
+find_package(HDF5 COMPONENTS CXX QUIET CONFIG)
+
+if(NOT HDF5_FOUND)
```

Comment:
If you're planning to role back this change later, please add a TODO or NOTE about this change